### PR TITLE
Library support

### PR DESCRIPTION
**Pull request by slowy07**
adding library support for terminal GUI

progress #6

### CODE REVIEW

File: kernel/library/color.asm [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    jz .invisible
+
+    dec al
+
+.invisible:
+    not al
+
+    mov byte[rsi + 0x03], al
+
+    dd rsi, KERNEL_VIDEO_DEPTH_shift
```
Comment (khairanabila) [inline kernel/library/color.asm]:
```asm
add rsi, KERNEL_VIDEO_DEPTH_shift
```
???

**Review by khairanabila — CHANGES_REQUESTED** [review]

**Comment by slowy07** [comment]
typo ;v

File: kernel/library/color.asm (hunk 1 above)
Comment (slowy07) [reply kernel/library/color.asm]: fixing, typo

**Review by khairanabila — APPROVED** [review]
LGTM

**Pull request merged**